internal/http/client.go

```diff
@@ -76,3 +121,17 @@ func (client Client) Do(method string, url string, headers RequestHeaders, body
 
 	return responseBody, ResponseHeaders{resp.Header}, nil
 }
+
+func withExponentialBackOff(retry int, base time.Duration, max time.Duration) time.Duration {
```

Comment:
This can probably be a method on our `Client` struct with jitter included

`func (c Client) retryDuration(retry int) time.Duration`